doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -154,19 +154,26 @@ Performing prerequisite operations
 
 You must enable the :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig option to support the Google Fast Pair standard in your project.
 
+The Fast Pair subsystem requires enabling Bluetooth with the :c:func:`bt_enable` function and loading Zephyr's :ref:`zephyr:settings_api` with the :c:func:`settings_load` function.
```

Comment:
Maybe we can put this section in the proximity of L171, as you don't need to perform these operations before calling register APIs. You just need to prepare these dependencies before you enable the Fast Pair subsystem:

```
After the callback registration, the Fast Pair subsystem must be enabled with the :c:func:`bt_fast_pair_enable` function.
Before the :c:func:`bt_fast_pair_enable` operation, you must enable Bluetooth with the :c:func:`bt_enable` function and load Zephyr's :ref:`zephyr:settings_api` with the :c:func:`settings_load` function.
```